packages/remark-language-server/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "remark-language-server",
+  "version": "1.0.0",
+  "description": "A language server for markdown using remark",
+  "homepage": "https://remark.js.org",
+  "repository": "https://github.com/remarkjs/remark/tree/main/packages/remark-language-server",
+  "bugs": "https://github.com/remarkjs/remark-language-serve/issues",
+  "bin": "remark-language-server.js",
+  "author": "Remco Haszing <remcohaszing@gmail.com>",
```

Comment:
Are you the copyright holder? Is this one of the packages (or does it include work of) originally other folks?